run-local-with-dev-db.sh

```diff
@@ -0,0 +1,43 @@
+#
```

Comment:
Is this your personal script - don't think we need it committing?

Reply:
it felt useful :)

Reply:
I think this could be useful, it could possibly have been an extension of the other script. 

The comment doesn't match what it does though.. it would be good to fix that and we could maybe look at consolidating the two at a later date.